worlds/tits_the_3rd/client/memory_io.py

```diff
@@ -122,35 +122,35 @@ def _get_scena_file_offsets(self, scena_file_path: str) -> List[int]:
             raise err
         return function_offsets
 
-    def _read_byte(self, offset):
+    def _read_bytes(self, offset, length=1):
         """
-        Read 1 byte of data at <base_process_address> + offset and return it.
+        Read length byte of data at <base_process_address> + offset and return it.
 
         offset (int): the offset to read data from.
         returns: The data represented as a byte string
         """
-        data = self.tits_the_3rd_mem.read_bytes(self.tits_the_3rd_mem.base_address + offset, 1)
+        data = self.tits_the_3rd_mem.read_bytes(self.tits_the_3rd_mem.base_address + offset, length)
         return data[0]
 
-    def _read_word(self, offset):
+    def _read_int(self, offset, byteorder: Literal["little", "big"] = "big"):
         """
-        Read 4 bytes of data at <base_process_address> + offset and return it.
+        Read 4 bytes at the specified offset, and interpret it as an int.
 
         offset (int): the offset to read data from.
-        returns: The data represented as a byte string
+        returns: The data represented as a float.
         """
-        data = self.tits_the_3rd_mem.read_bytes(self.tits_the_3rd_mem.base_address + offset, 4)
-        return data
+        data = self._read_bytes(offset, 4)
+        return int.from_bytes(data, byteorder)
 
-    def _read_int(self, offset):
+    def _read_short(self, offset, byteorder: Literal["little", "big"] = "big"):
         """
-        Read a word at the specified offset, and interpret it as an int.
+        Read 2 bytes at the specified offset, and interpret it as an int.
 
         offset (int): the offset to read data from.
         returns: The data represented as a float.
         """
-        data = self._read_word(offset)
-        return struct.unpack("i", data)[0]
+        data = self._read_bytes(offset, 2)
+        return int.from_bytes(data, byteorder)
```

Comment:
What's the use case for allowing little endian?

Reply:
The game actually uses little endian for most things and it's actually rarer to see big endian

Reply:
Do we need to make little endian the default here then?